src/components/Menu/components/SettingsModal/SettingsModal.js

```diff
@@ -0,0 +1,36 @@
+import { Modal, StyleSheet, Text, Pressable, View, Image, TouchableHighlight } from "react-native";
+import styles from "./styles";
+
+const removeIcon = require("../../../../img/icons/remove_icon.png");
+
+const SettingsModal = ({ modalVisible, setModalVisible }) => {
+  return (
+    <Modal
+      animationType="slide"
+      transparent={true}
+      visible={modalVisible}
+      onRequestClose={() => {
+        setModalVisible(!modalVisible);
+      }}
+    >
+      <View style={styles.centeredView}>
+        <View style={styles.modalView}>
+          <TouchableHighlight
+            style={styles.remove}
+            onPress={() => setModalVisible(!modalVisible)}
```

Comment:
What if I will click it twice fast enough?) If you want to close it - use need to close it, instead of toggling it's value